apps/nextjs-website/src/helpers/productHeader.helper.ts

```diff
@@ -13,7 +13,7 @@ export function productToMenuItems(
       return {
         label: subpath.name,
         href: subpath.path,
-        active: path === subpath.path,
+        active: path.includes(subpath.path),
```

Comment:
Could you add a unit test that catches this issue?

If we want to be more sure to not having troubles with paths I'd use the `startsWith` method, what do you think?
The corner case is:
`path` is `/a/b/c-d/e` and `subpath.path` is `/b/c-d`